### PR TITLE
feat: like page added

### PR DESCRIPTION
**Pull request by mehulsatardekar**
Live preview 👍  : https://velvety-cranachan-b64fc8.netlify.app/likeplaylist

files to check: 👍 
src/UI/Pages/LikeVideoPage/LikeVideoPage.jsx
 create mode 100644 src/UI/Pages/LikeVideoPage/like-video-page.css
 create mode 100644 src/components/Like-videos/LikeVideos.jsx

### CODE REVIEW

File: src/components/Like-videos/LikeVideos.jsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  const deleteVideo = async(movieid, video)=>{
+     const data = await deleteLikeVideo(movieid);
+
+     if(data.status===200){
+      likeVideoDispatch({type:'DELETE_LIKE_VIDEO', payload:video })
+     }
+     console.log('deleted data',data)
+  }
+
+  console.log(likeVideoState.likedVideos);
```
Comment (gauravsinhaweb) [inline src/components/Like-videos/LikeVideos.jsx:22]: remove this log

**Review by gauravsinhaweb — APPROVED** [review]
It's good to keep the review file short!! Good to merge

File: src/components/Like-videos/LikeVideos.jsx (hunk 1 above)
Comment (mehulsatardekar) [reply src/components/Like-videos/LikeVideos.jsx:22]: sure will do that

File: src/components/Like-videos/LikeVideos.jsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { viewsFormatter } from "../../utils/views-formatter";
+import { deleteLikeVideo } from "../../utils";
```
Comment (prath-dev) [inline src/components/Like-videos/LikeVideos.jsx:4]: use barrel import export .. just like you did it on 4 line

**Review by prath-dev — APPROVED** [review]
looking good nice work on ui

**Pull request merged**